guides/user/canary/config/index.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: single
+title:  "Configure Canary for an application"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Before you can add a canary stage to a pipeline, you need to configure what the
+canary consists of, including...
+
+* The specific metrics to evaluate, and a logical grouping of those metrics
+* Default scoring thresholds (which can be overridden in the
+canary stage)
+* Optionally, one or more filter templates
+* A name by which a canary stage can choose this config
+
+Canary configuration is done per Spinnaker
+[application](/concepts/#applications). For each
+application set up to support canary, you create one or more configs. Your
+setup might even be that all configurations are available to all
+applications.
+
+The configuration you create here is available to canary stages in pipelines,
+but those [stages are defined separately](guides/user/canary/stage/).
+
+<!---
+TODO: figure out why this is done and how it's used
+--->
+Separately, you can [group individual metrics by Stackdriver
+resource types](#group_the_metrics).
+
+## Prerequisites
+
+By default, canary is not enabled for new applications. Several things need to
+happen before you see the __Canary__ tab in Deck:
+
+* The person setting up Spinnaker for you must [set up Canary](/setup/canary/).
+
+  This includes specifying whether all canary configs are available to all
+  applications or each application can have one or more configurations that are
+  visible to that application only.
+
+* In the Application config, activate the __Canary__ option.
+
+  Do this separately for all applications that will use automated canary
+  analysis.
+
+{%
+ include
+ figure
+ image_path="./enable_canary.png"
+%}
+
+## Create a canary configuration
+
+You can create as many of these as you like, and when you create a canary stage,
+you can select which configuration to use. Configurations you create within an
+application are available to all pipelines in that application, but your
+Spinnaker might be set up so that all configurations are available to all
+pipelines.
```

Comment:
@dorbin 
your Spinnaker might be set up so that all configurations are available to all applications (currently says  all pipelines).

Reply:
Thanks, @t0mpson !